src/agent/communicator/test/communicator_test.cpp

```diff
@@ -0,0 +1,38 @@
+#include "communicator.hpp"
+#include <boost/asio/connect.hpp>
```

Comment:
Looks like most of these headers are not being used.

Reply:
Done